cli/ballerina-packerina/src/test/java/org/ballerinalang/packerina/cmd/BuildCommandTest.java

```diff
@@ -287,6 +287,22 @@ public void testBuildAllWithArg() throws IOException {
         Assert.assertTrue(buildLog.contains("too many arguments.\n"));
     }
 
+    @Test(description = "Build .bal files with the relative path")
+    public void testBuildWithRelativePath() throws IOException {
+        // valid source root path
+        Path validBalFilePath = this.testResources.resolve("valid-bal-file");
+        BuildCommand buildCommand = new BuildCommand(validBalFilePath, printStream, printStream, false, true);
+        new CommandLine(buildCommand).parse(String.valueOf(validBalFilePath.resolve
+                ("valid-relative-path-bal-file/hello_world_relative.bal")));
```

Comment:
This wont give a relative path right ? `validBalFilePath.resolve("valid-relative-path-bal-file/hello_world_relative.bal")`

Also we cannot have the file separate as `/`. Wont it break in Windows ?

Reply:
Modified with proper relative path.